### PR TITLE
Update mods.json

### PR DESCRIPTION
**Pull request by Trinsdar**

### CODE REVIEW

File: data/mods.json [hunk 1]
```diff
@@ -107,6 +107,16 @@
...
 	},
+	{
+		"name": [
+			"IC2",
+			"ic2"
+		],
+		"versions": [
+			"1.12.2"
+		],
+		"repository": "Trinsdar"
```
Comment (RobertRR11) [inline data/mods.json:118]: please change "repository" back to Faithful-Mods

**Comment by RobertRR11** [comment]
Thanks for contributing to Faithful Mods! It looks like you have more than just resource pack files on your resource pack repository. When we possibly transfer the repository to the organisation we may delete all other files not related to the resource pack, including the license if you agree.

File: data/mods.json [hunk 2]
```diff
@@ -107,6 +107,16 @@
 		],
 		"repository": "Faithful-Mods"
 	},
+	{
+		"name": [
+			"IC2",
```
Comment (Juknum) [inline data/mods.json:112]: better: Industrial Craft 2 (IC2)

File: data/mods.json (hunk 1 above)
Comment (Juknum) [reply data/mods.json:118]: only if the repository is merged

**Review by Juknum — COMMENTED** [review]
Awesome!

**Comment by Juknum** [comment]
> Thanks for contributing to Faithful Mods! It looks like you have more than just resource pack files on your resource pack repository. When we possibly transfer the repository to the organisation we may delete all other files not related to the resource pack, including the license if you agree.

Maybe we can leave the License in it, like this, when people compile the pack, they get the license of ic2 🤔

**Comment by Trinsdar** [comment]
the license would need to stay in, cause not all the textures were done by me

**Comment by Trinsdar** [comment]
and I didn't delete the presets folder cause I haven't figured out where that went to yet

File: data/mods.json (hunk 1 above)
Comment (RobertRR11) [reply data/mods.json:118]: whoops

**Pull request merged**